src/backends/gds.jl

```diff
@@ -10,7 +10,7 @@ import ..Rectangles: Rectangle
 import ..Polygons: Polygon
 using ..Cells
 
-import FileIO: File, @format_str, load, save, stream, magic, skipmagic
+import FileIO: File, @format_str, load, stream, magic, skipmagic
```

Comment:
Can we also do the same for `load`? My understanding is that the FileIO interface also requires that we don't implement `FileIO.load`

Reply:
We should be able to. Do we ever end up loading GDS files?

Reply:
Yes, it's useful to have.